source/reference/api/reverse.txt

```diff
@@ -43,6 +43,8 @@ To use the ``reverse`` endpoint:
   
   You cannot update these options after the sync starts.
 - ``mongosync`` must be in the ``COMMITTED`` state.
+- The destination cluster oplog must not roll over before
+  ``mongosync`` reaches the ``COMMITTED`` state.
```

Comment:
This isn't quite right. The requirement is that the destination cannot roll over _after_ mongosync commits if the user intends to reverse in the future. A different way of phrasing this is:
> The destination cluster oplog must not roll over between `mongosync` reaching the `COMMITTED` state and `mongosync` receiving a `reverse` request.

Reply:
Fixed